packages/source-iottwinmaker/package.json

```diff
@@ -63,7 +62,9 @@
     "rxjs": "^7.8.1"
   },
   "devDependencies": {
-    "@aws-sdk/types": "^3.310.0",
+    "@aws-sdk/client-iottwinmaker": "file:~/workplace/AppKitV5/aws-sdk-client-iottwinmaker-3.571.0.tgz",
```

Comment:
Maybe this PR should be set to `Draft` until this is dealt with?

Reply:
Good idea.